lib/cli/scripts/plugins/aps-plugin.ts

```diff
@@ -0,0 +1,61 @@
+import { PluginInterface } from './plugin-model';
+import { logger } from '../logger';
+import { PluginConfiguration } from './plugin-config';
+
+export class ProcessServicePlugin {
```

Comment:
can we rename the file into process-service-check-plugin?

Reply:
Updated